signals.go

```diff
@@ -41,17 +41,10 @@ func signalHandler(listener net.Listener, statusListener net.Listener, stopper c
 	for {
 		// Wait for a signal
 		select {
-		case sig := <-signals:
-			switch sig {
-			case syscall.SIGTERM:
-				logger.Printf("received SIGTERM, stopping listener")
+		case _ = <-signals:
```

Comment:
Hmm, I thought we handled SIGTERM to do graceful shutdown. Can you clean up the comment above referring to SIGTERM? Otherwise LGTM.